rocAL/rocAL/include/augmentations/geometry_augmentations/node_crop_mirror_normalize.h

```diff
@@ -25,26 +25,23 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_crop_factory.h"
 #include "parameter_vx.h"
+
 class CropMirrorNormalizeNode : public Node
 {
 public:
-    CropMirrorNormalizeNode(const std::vector<Image *> &inputs,
-                            const std::vector<Image *> &outputs);
+    CropMirrorNormalizeNode(const std::vector<rocalTensor *> &inputs,
+                            const std::vector<rocalTensor *> &outputs);
     CropMirrorNormalizeNode() = delete;
-    void init(int crop_h, int crop_w, float start_x, float start_y, float mean, float std_dev, IntParam *mirror);
+    void init(int crop_h, int crop_w, float start_x, float start_y, std::vector<float>& mean,  std::vector<float>& std_dev, IntParam *mirror);
     vx_array return_mirror(){ return _mirror.default_array();  }
     std::shared_ptr<RocalCropParam> return_crop_param() { return _crop_param; }
-    vx_array get_src_width() { return _src_roi_width; }
-    vx_array get_src_height() { return _src_roi_height; }
 protected:
     void create_node() override ;
     void update_node() override;
 private:
     std::shared_ptr<RocalCropParam> _crop_param;
     std::vector<vx_float32> _mean_vx, _std_dev_vx;
```

Comment:
The usage of  _mean_vx, _std_dev_vx seems to be removed in the .cpp files.
Remove this if not used

Reply:
Done